.github/workflows/docker.yaml

```diff
@@ -39,30 +39,57 @@ jobs:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
 
-    - name: 🔨 Build and push
+    - name: 📝 Base Variables
+      id: base-variables
+      run: |
+        echo ::set-output name=image::cardanosolutions/${{ matrix.target }}
```

Comment:
There isn't much benefits defining a variable for this one. On [hydra-poc](https://github.com/input-output-hk/hydra-poc/blob/master/.github/workflows/docker.yaml#L13) I ended up configuring the repository in the build matrix itself to use as such:

```yaml
[...]
        cache-from: type=registry,ref=${{ matrix.repository }}:${{ matrix.target }}-latest
[...]
```

But, it doesn't really cost much either to have a variable defined so :man_shrugging:

Reply:
Ah, that's clean. I'll update it

Reply:
Ah that's clean, I'll update next time